Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -42,6 +44,8 @@ @interface CDTURLSessionTask ()
 
 @property (nonatomic) int remainingRetries;
 
+@property BOOL finished;
```

Comment:
Properties are atomic by default, but I guess there's no harm being explicit about it. Updated in 3f6b6ec.